modules/statistics/test/TestPlan.md

```diff
@@ -30,6 +30,14 @@
 13. Returning to the behavioural statistics page, verify that if you do not have the `access_all_profiles` permission you should not be able to click through to forms for candidates from other sites.
 14. Check the breadcrumbs from Completion stats page back to BVL stats tab within Statistics appear and allow you to use the `Back` button to previous pages (future feature).
 
+### Dashboard Widget
+1. Go to the LORIS dashboard. 
+     - Ensure that the "My Tasks" on the LORIS dashboard has a "Incomplete Forms"
```

Comment:
Isn't that for the conflict resolver, not the statistics module?

The statistics module provides the study progression charts..

Reply:
@driusan I don't know if I misunderstand something. Conflict Resolver is associated with Data Conflicts item, right? and Incomplete Forms to Behavioural statistics.

Reply:
sorry, Incomplete Forms comes from the instruments module.. still not statistics.